### PR TITLE
qa/tests: added v14.2.1

### PR DESCRIPTION
**Pull request by yuriw**
Fixes: http://tracker.ceph.com/issues/40181
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2019-06-05_22:22:46-upgrade:nautilus-p2p-nautilus-distro-basic-smithi/

**Comment by yuriw** [comment]
@jdurgin see ""2019-06-05 23:39:45.244049 osd.0 (osd.0) 1 : cluster [WRN] failed to encode map e761 with expected crc" in cluster log"

can we whitelist `failed to encode map`?

**Comment by yuriw** [comment]
Per @jdurgin "the encode map" .. "may be a real problem between those releases"

So standing by

**Comment by yuriw** [comment]
whitelisted "failed to encode map"

http://pulpito.front.sepia.ceph.com:80/yuriw-2019-06-10_22:23:08-upgrade:nautilus-p2p-nautilus-distro-basic-smithi/

File: qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ overrides:
     - cache pools at or near target size
     - filesystem is degraded
     - OBJECT_MISPLACED
+    - failed to encode map
```
Comment (neha-ojha) [inline qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml]: let's add a reference to https://tracker.ceph.com/issues/40251 here

Comment (yuriw) [reply qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml]: done, hope it won't break yaml parser :)

**Comment by jdurgin** [comment]
looks good

**Comment by yuriw** [comment]
See http://tracker.ceph.com/issues/40304 for failures refs

**Pull request merged**